src/components/try-hcp-callout/components/try-hcp-callout-compact/try-hcp-callout-compact.module.css

```diff
@@ -24,7 +24,7 @@
 
 .background {
 	composes: backgroundBase;
-	background-image: url('./img/try-hcp-callout-compact-background.svg');
+	background-image: url('./img/try-hcp-callout-compact-background.svg?url');
```

Comment:
`?url` suffix here ensures the SVG background doesn't get loaded in as a data string, which should help bundle size as well.